### PR TITLE
feat(solana): Add method for finding solana deposit signature [WEB-1677]

### PR DESCRIPTION
**Pull request by niklasnatter**

### CODE REVIEW

File: packages/solana/src/__test__/deposit.test.ts [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+        ],
+        addressTableLookups: [],
+      }),
+      signatures: [
+        '3zC67SpVaUAMXitSZw6Z1HtTPFtEEeKdJ2cAkHGAPYyBdWkbbT3RRva9Do6SbQjk8Zs8oHJrprV8txdc3YgBp33x',
+      ],
+    },
+    version: 'legacy',
+  },
+  '': {
```
Comment (acdibble) [inline packages/solana/src/__test__/deposit.test.ts:83]: probably not needed?

**Review by acdibble — APPROVED** [review]

File: packages/solana/package.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
     "test": "vitest"
   },
   "dependencies": {
-    "@chainflip/utils": "0.4.0"
+    "@chainflip/utils": "0.4.0",
+    "@solana/web3.js": "^1.95.8"
```
Comment (zoheb391) [inline packages/solana/package.json:18]: this is the not affected version right. just checking :D

**Review by zoheb391 — APPROVED** [review]
reading this PR made me realise that i know fuck all about whats happening on solana

File: packages/solana/package.json (hunk 2 above)
Comment (niklasnatter) [reply packages/solana/package.json:18]: no, but i actually managed to install the affected version when i started with this pr and was wondering why the CI does not work 😅

File: packages/solana/src/__test__/deposit.test.ts (hunk 1 above)
Comment (niklasnatter) [reply packages/solana/src/__test__/deposit.test.ts:83]: good catch! its not strictly needed but we want to return it from one of the `getTransactions` calls to have a realistic spam test

**Pull request merged**